migrate.sh

```diff
@@ -3,241 +3,31 @@
-echo "pkg encoding/json/v2, type Unmarshaler interface { UnmarshalJSON } #$ISSUE" >> $NEXT
-echo "pkg encoding/json/v2, type Unmarshaler interface, UnmarshalJSON([]uint8) error #$ISSUE" >> $NEXT
-echo "pkg encoding/json/v2, type UnmarshalerFrom interface { UnmarshalJSONFrom } #$ISSUE" >> $NEXT
-echo "pkg encoding/json/v2, type UnmarshalerFrom interface, UnmarshalJSONFrom(*jsontext.Decoder, jsonopts.Options) error #$ISSUE" >> $NEXT
-echo "pkg encoding/json/v2, type Unmarshalers = typedArshalers[jsontext.Decoder] #$ISSUE" >> $NEXT
-echo "pkg encoding/json/v2, var ErrUnknownName error #$ISSUE" >> $NEXT
-echo "pkg encoding/json/v2, var SkipFunc error #$ISSUE" >> $NEXT
-# The following declarations were moved to encoding/json/v2 or encoding/json/jsontext.
-EXCEPT="$GOROOT/api/except.txt"
-echo "pkg encoding/json, method (*RawMessage) UnmarshalJSON([]uint8) error" >> $EXCEPT
-echo "pkg encoding/json, method (RawMessage) MarshalJSON() ([]uint8, error)" >> $EXCEPT
-echo "pkg encoding/json, type Marshaler interface { MarshalJSON }" >> $EXCEPT
-echo "pkg encoding/json, type Marshaler interface, MarshalJSON() ([]uint8, error)" >> $EXCEPT
-echo "pkg encoding/json, type RawMessage []uint8" >> $EXCEPT
-echo "pkg encoding/json, type Unmarshaler interface { UnmarshalJSON }" >> $EXCEPT
-echo "pkg encoding/json, type Unmarshaler interface, UnmarshalJSON([]uint8) error" >> $EXCEPT
-
-# Run the tests.
-(cd $GOROOT/src; ./all.bash)
+/bin/rm -r $JSONROOT/*.go $JSONROOT/internal $JSONROOT/jsontext $JSONROOT/v1
+cp -r $GOROOT/src/encoding/json/v2/*.go $JSONROOT/
+cp -r $GOROOT/src/encoding/json/internal/ $JSONROOT/internal/
+cp -r $GOROOT/src/encoding/json/jsontext/ $JSONROOT/jsontext/
+mkdir $JSONROOT/v1
+for X in $GOROOT/src/encoding/json/v2_*.go; do
+    cp $X $JSONROOT/v1/$(basename $X | sed "s/v2_//")
+done
+cd $JSONROOT
+for X in $(git ls-files --cached --others --exclude-standard | grep ".*[.]go$"); do
+    if [ ! -e "$X" ]; then
+        continue
+    fi
+    sed -i '/go:build goexperiment.jsonv2/d' $X
+    sed -i 's|"encoding/json/v2"|"github.com/go-json-experiment/json"|' $X
+    sed -i 's|"encoding/json/internal"|"github.com/go-json-experiment/json/internal"|' $X
+    sed -i 's|"encoding/json/internal/jsonflags"|"github.com/go-json-experiment/json/internal/jsonflags"|' $X
+    sed -i 's|"encoding/json/internal/jsonopts"|"github.com/go-json-experiment/json/internal/jsonopts"|' $X
+    sed -i 's|"encoding/json/internal/jsontest"|"github.com/go-json-experiment/json/internal/jsontest"|' $X
+    sed -i 's|"encoding/json/internal/jsonwire"|"github.com/go-json-experiment/json/internal/jsonwire"|' $X
+    sed -i 's|"encoding/json/jsontext"|"github.com/go-json-experiment/json/jsontext"|' $X
+    sed -i 's|"encoding/json"|"github.com/go-json-experiment/json/v1"|' $X
+    sed -i 's|"internal/zstd"|"github.com/go-json-experiment/json/internal/zstd"|' $X
+    goimports -w $X
+done
+sed -i 's/v2.struct/json.struct/' $JSONROOT/errors_test.go
```

Comment:
nit: did you mean to use `\.` so that it doesn't match e.g. `v2_struct`?

Reply:
Yea, but it seemed to work :P. I'll fix it.